Misc/NEWS.d/next/Core and Builtins/2018-02-16-10-44-24.bpo-32856.UjR8SD.rst

```diff
@@ -0,0 +1,3 @@
+Optimized the idiom for assignment a temporary variable in comprehensions.
+Now ``for y in [expr]`` in comprehensions is so fast as a simple assignment
+``y = expr``.
```

Comment:
Please update the NEWS entry with some benchmark results.

Reply:
I'm not sure what should I provide. The optimized code always is the part of complex comprehension expression.